modules/mobile/docs/schemas/Appointments.yml

```diff
@@ -2,7 +2,6 @@ type: object
 required:
   - data
   - meta
-  - links
```

Comment:
Seems like we're pretty inconsistent in how we handle this. Sometimes we added links, sometimes not. Sometimes we add meta, sometimes not. Maybe worth a ticket to just double check and clean up this documentation since things are pretty quiet right now.